packages/docs/src/content/03-components/content/image/overview.mdx

```diff
@@ -1,3 +1,13 @@
 # Beispiel
 
 <LiveCodeEditor />
+
+---
+
+# Responsive Image Resolution
+
+Um unterschiedliche Bild-Auflösungen für diverse Bildschirmgrößen anzubieten
+eignet sich das `srcSet` Attribut. Für diverse Pixeldichten auf Nutzerseite
+können hiermit verschiedene niedrig- oder hochauflösende Varianten der
+Quelldatei hinterlegt werden. Näheres hierzu ist in der React Doku zu
+[srcSet](https://reactnative.dev/docs/image#srcset) einsehbar.
```

Comment:
Thanks for your contribution. Is it intended that linked the React _Native_ docs? What about the mdn web docs? I usually refer them in the matter of standard web technology: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images

Reply:
Link has been changed